modules/ldap-syncing-pruning.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing.adoc
+
+[id="ldap-syncing-pruning_{context}"]
+= Running a group pruning job
```

Comment:
This section kinda seems incomplete. Should there be a procedure below it to show how to run a group pruning job?